### PR TITLE
OCPBUGS-46497:adding new NVIDIA accelerators to monitor

### PR DESCRIPTION
**Pull request by yevgeny-shnaidman**

### CODE REVIEW

**Comment by yevgeny-shnaidman** [comment]
/retitle OCPBUGS-46497:adding new NVIDIA accelerators to monitor

**Comment by juzhao** [comment]
I think we should fix in 4.19 first, then we could backport to 4.18, although there is way to merge 4.18 first

**Review by simonpasquier — COMMENTED** [review]
/lgtm
/approve

File: collector/accelerators.go [hunk 1]
```diff
@@ -84,6 +84,14 @@ var (
 		"0x2233": "NVIDIA RTX A5500",
 		"0x1e30": "NVIDIA RTX 8000 passive",
 		"0x2531": "NVIDIA RTX A2000",
+		"0x20b0": "NVIDIA A100 SXM4 40G",
```
Comment (juzhao) [inline collector/accelerators.go:87]: mabye
`"0x20b0": "NVIDIA A100 SXM4 40GB",`

File: collector/accelerators.go [hunk 2]
```diff
@@ -84,6 +84,14 @@ var (
...
+		"0x233a": "NVIDIA H800 NVL",
+		"0x233b": "NVIDIA H200 NVL",
+		"0x20b2": "NVIDIA A100SXM4 80GB",
```
Comment (juzhao) [inline collector/accelerators.go:90]: "0x20b2": "NVIDIA A100 SXM4 80GB",

File: collector/accelerators.go (hunk 1 above)
Comment (yevgeny-shnaidman) [reply collector/accelerators.go:87]: @juzhao, this seems like a very minor issue in description. I won't fixed here, but i will fix it in the future versions

**Pull request merged**